packages/shared/src/components/squads/SquadHeaderBar.tsx

```diff
@@ -39,20 +51,28 @@ export function SquadHeaderBar({
         members={members}
         memberCount={memberCount}
       />
-      <SimpleTooltip placement="top" content="Squad options">
+      <span className="flex flex-row gap-4">
+        <Button className="btn-secondary" onClick={openSquadInviteModal}>
+          Invite
+        </Button>
         <Button
-          className="tablet:order-2 btn btn-secondary"
-          icon={<MenuIcon size={IconSize.Small} />}
-          onClick={onMenuClick}
-          aria-label="Squad options"
+          tag="a"
+          target="_blank"
+          rel="noopener noreferrer"
+          href={`${squadFeedback}#user_id=${user.id}&squad_id=${squad.id}`}
+          className="btn-secondary"
+          icon={<FeedbackIcon aria-label="squad-feedback-icon" />}
+          aria-label="squad-feedback"
         />
```

Comment:
We should probably wrap this in a tooltip now as it's not super descriptive what the button does.

Reply:
Totally agree with that.